app/controllers/users_controller.rb

```diff
@@ -7,6 +7,10 @@ def index
     @users = User.all
   end
 
+  def defualt
```

Comment:
__[staging][Misspell]__

"defualt" is a misspelling of "default"


<!-- Comment by [Sider](https://sider.review) -->